src/sparql/omim-relevant-diseases.sparql

```diff
@@ -40,5 +40,5 @@ WHERE {
     }
     
   }
-  FILTER(isIRI(?term) && (regex(str(?term), "https://omim.org/entry/") || regex(str(?term), "https://www.omim.org/phenotypicSeries/")))
+  FILTER(isIRI(?term) && (regex(str(?term), "https://omim.org/entry/") || regex(str(?term), "https://omim.org/phenotypicSeries/")))
```

Comment:
## Changes
This change and the change in the other SPARQL file are basically identical.

Please also see the discussion in this comment for (i) background on this issue, and (ii) general plan:
- https://github.com/monarch-initiative/omim/pull/109#discussion_r1582496380